core/templates/pages/landing-pages/topic-landing-page/topic-landing-page.controller.spec.ts

```diff
@@ -124,16 +87,6 @@ describe('Topic Landing Page', function() {
     ctrl.onClickLearnMoreButton();
     $timeout.flush(150);
 
-    expect(windowRef.nativeWindow.location).toBe('/');
-  });
-
-  it('should click exploration lessons button', function() {
```

Comment:
Why are you removing these specs?

Reply:
These are no more in use and we have a simple landing page now!

Reply:
Thanks for the explanation!